basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/HTTPSListenerTestCase.java

```diff
@@ -363,7 +365,17 @@ public void editSendBufferInvalid() throws Exception {
 
     @Test
     public void editSocketBinding() throws Exception {
-        editTextAndVerify(HTTPS_LISTENER_ADDRESS, SOCKET_BINDING, undertowOps.createSocketBindingWithoutReference());
+        String socketBinding = undertowOps.createSocketBinding();
+        ModelNodeResult result = operations.readAttribute(HTTPS_LISTENER_ADDRESS, SOCKET_BINDING);
```

Comment:
Also using more descriptive name of variable for the returned value would be nice :-)